psalm.xml

```diff
@@ -32,6 +32,11 @@
 				<referencedProperty name="Sensei_Main::$lesson_progress_repository_factory" />
 				<referencedProperty name="Sensei_Main::$quiz_progress_repository_factory" />
 				<referencedProperty name="Sensei_REST_API_Lesson_Actions_Controller::$schema" />
+				<referencedProperty name="Sensei_PostTypes::$course" />
+				<referencedProperty name="Sensei_PostTypes::$lesson" />
+				<referencedProperty name="Sensei_PostTypes::$question" />
+				<referencedProperty name="Sensei_PostTypes::$quiz" />
+				<referencedProperty name="Sensei_PostTypes::$messages" />
```

Comment:
This is needed because the constructor properties are set [dynamically](https://github.com/Automattic/sensei/blob/fix/early-translations/includes/class-sensei-posttypes.php#L100).